tests/copp/conftest.py

```diff
@@ -58,3 +58,14 @@ def is_backend_topology(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbi
     is_backend_topology = mg_facts.get(constants.IS_BACKEND_TOPOLOGY_KEY, False)
 
     return is_backend_topology
+
+
+@pytest.fixture(params=[64, 1518, 4096])
```

Comment:
@senthil-nexthop could you please clarify if you meant 1514 here? The PR description says packets > 1514 need special handling, so 1518 and 4096 would be testing similar things right? If the PR description is correct, could you change this to 1514 instead of 1518?